static/css/section/_mobile.scss

```diff
@@ -1,40 +1,84 @@
-.mobile-features {
-  .row-hero {
-    overflow: hidden;
-
-    @media only screen and (min-width : $breakpoint-medium) {
+.mobile- {
```

Comment:
Here you're creating a class called `.mobile-`. I don't know if that's what you intended. If not, I think you should instead have top-level classes with the full names you want to use, e.g. `.mobile-features {}`, `.mobile-partners {}`. Or if you want, you could have a top-level class called `.mobile {}` and then put `&__features {}` inside it, but I'd only suggest doing that if you were actually going to use the `.mobile` class.